### DISCUSSION TITLE
IDH Config

### DISCUSSION THREAD

**Discussion by lugashisnotafed** [Security-Onion-Solutions/securityonion#7551] [Unsupported Versions] [answered]
Hey,
I've been playing around with the IDH function. 
Was wondering if there's a way to  have "servers" on a different eth adapter . 
In this deployment the manager is off prem and we send all the data offsite so I usually create a tunnel on my management interface for the data stuffs and then use the other interface for sniffing etc.

The way I currently see IDH being deployed is that it has to be able to hit the Manager from the same interface that it is running the 'server' on.

Any thoughts/suggestions/pointing at a place in the docs would be appreciated.

**Comment by defensivedepth** [comment, marked as answer]
Would the ability to have a `Management` NIC and one or more `Honeypot` NIC solve your use case? To be clear, there would still be one honeypot on the IDH node, just that it could bind to one or more NICs, separate from the Management interface for the box (which would need to be able to contact the Grid Manager)

**Reply by lugashisnotafed** [reply]
Yea! that would be amazing.

**Reply by defensivedepth** [reply]
I have created an issue to track this: https://github.com/Security-Onion-Solutions/securityonion/issues/7589

**Reply by defensivedepth** [reply]
@lugashisnotafed Something like this?

![image](https://user-images.githubusercontent.com/954732/161819261-5fc500df-1f6e-48af-81b0-f3f276832889.png)

**Reply by lugashisnotafed** [reply]
Or just have the ability to first choose the management interface like in regular sensor installs and then have the ability to choose the other interfaces to run the honeypot node on.

**Reply by defensivedepth** [reply]
How likely is it that you would have IPs configured on an IDH node (other than the Management IP), that you wouldn't want to publish the IDH services on?

**Reply by lugashisnotafed** [reply]
extermely unlikey i think.
